src/Symfony/Component/Validator/Constraints/DateTimeValidator.php

```diff
@@ -68,6 +68,8 @@ public function validate($value, Constraint $constraint)
                     ->setParameter('{{ value }}', $this->formatValue($value))
                     ->setCode(DateTime::INVALID_TIME_ERROR)
                     ->addViolation();
+            } elseif ('Trailing data' === $warning && '+' === substr($constraint->format, -1)) {
```

Comment:
Could there be cases with more than one warning where don't want to ignore the other ones?

Reply:
@stefankleff any idea? Could you please have a look?

Reply:
@stefankleff Have you time to have a look at this?

Reply:
Sorry, not yet. But I'll try within this week.

Reply:
friendly ping

Reply:
The case is valid, that we have a "+" in the format and we don't want to ignore the other warnings.
I couldn't find a better way than remove the warning from the list of warnings.
I also looked in the underlying c-code and I couldn't find any similar cases to trailing data.